lib/common/index.js

```diff
@@ -38,4 +39,9 @@ for (i = 0; i < vmcommonFunctions.length; i++) {
 var predicateFunctions = Object.keys(predicate);
 for (i = 0; i < predicateFunctions.length; i++) {
     exports[predicateFunctions[i]] = predicate[predicateFunctions[i]];
-}
\ No newline at end of file
+}
+
+var markerFunctions = Object.keys(marker);
+for (i = 0; i < markerFunctions.length; i++) {
+    exports[markerFunctions[i]] = marker[markerFunctions[i]];
```

Comment:
Good point, I factored that out into a `mixinModule` function that does this (not tested or committed/pushed yet):

```
function mixinModule(modulePath) {
    assert.string(moduleName, 'moduleName must be a string');

    var i;
    var module = require(modulePath);
    var moduleExports = Object.keys(module);

    for (i = 0; i < moduleExports.length; i++) {
        assert.equal(exports[moduleExports[i]], undefined, moduleExports[i] +
            ' from module ' + modulePath +
            ' must not already exist in target module ' + __filename);
        exports[moduleExports[i]] = moduleExports[moduleExports[i]];
    }
}
```